src/app/clusters/smoke-co-alarm-server/smoke-co-alarm-server.cpp

```diff
@@ -55,28 +55,28 @@ void SmokeCoAlarmServer::SetExpressedStateByPriority(EndpointId endpointId,
         switch (priority)
         {
         case ExpressedStateEnum::kSmokeAlarm:
-            VerifyOrReturn(GetSmokeState(endpointId, alarmState));
+            GetSmokeState(endpointId, alarmState);
```

Comment:
why does this change update to ignore the return value for these methods?

At least explicitly putting `(void)` in front to make readers aware that the return value was purposfully ignored ... however PR log should explain why that was done.

Reply:
I updated the PR log and changed it back to the original VerifyOrReturn, but it now allows for unsupported property errors.